src/stapi_fastapi/routers/root_router.py

```diff
@@ -306,9 +298,11 @@ def order_statuses_link(self, request: Request, order_id: str):
             type=TYPE_JSON,
         )
 
-    def pagination_link(self, request: Request, pagination_token: str):
+    def pagination_link(self, request: Request, pagination_token: str, limit: int):
         return Link(
-            href=str(request.url.include_query_params(next=pagination_token)),
+            href=str(
```

Comment:
include the limit explicitly, so it's not just relying on whatever the default is